.github/workflows/release_parallel.yml

```diff
@@ -111,85 +98,43 @@ jobs:
               continue-on-error: true
               run: npm test
 
-            - uses: ./.github/actions/configure_git
+            - name: Create branch name
+              id: create-branch-name
+              run: |
+                  SAFE_PATH=$(echo '${{ matrix.test.displayPath }}' | sed 's/\//-/g')
+                  echo "branch-name=update--${{ needs.prepare-release.outputs.release-version }}-${SAFE_PATH}" >> $GITHUB_OUTPUT
+              shell: bash
+
+            - uses: ./.github/actions/commit_and_push
               with:
+                  branch-name: release--${{ needs.prepare-release.outputs.release-version }}
+                  commit-message: 'Update dependencies for ${{ matrix.test.displayPath }}'
```

Comment:
Look into improving the name of the commit message

Reply:
done